modules/hydra-slave.nix

```diff
@@ -7,11 +9,28 @@ in
   services = {
     dd-agent.tags              = [" group:hydra-and-slaves"];
   };
+  nixpkgs.overlays = [
+    (self: super: {
+      nix' = super.nix.overrideAttrs ( drv: {
+        patches = [
+          (pkgs.fetchpatch {
+             url = "https://github.com/NixOS/nix/commit/a6c5955c8d59ca26745843f9779253a173876665.patch";
```

Comment:
I think it's worth mentioning why this is needed, as a comment -- so we have context on when to remove this overlay.